### PR TITLE
zuul: easy update procedure

### PR DESCRIPTION
**Pull request by sodar**
Documentation for easier Zuul update procedure. By easier I mean, it depends solely playbooks on `contrail-windows-ci` and does not require manual patching.

### CODE REVIEW

File: doc/ci_admin_guide/Update_Zuul.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Update Windows CI Zuulv2
+
+This document describes a procedure required to update Windows CI Zuulv2 configuration.
+This procedure updates configuration using `development` branch from [contrail-windows-ci][c-w-ci] repository.
+
+## Prerequisites
+
+- Ubuntu 16.04 or Windows Subsystem for Linux with Ubuntu
+- User's public SSH key installed on Zuulv2 instance
+    - Please contact with Windows CI team
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: > Please contact ~~with~~ Windows CI team

File: doc/ci_admin_guide/Update_Zuul.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Access to ansible vault key
+    - Please contact with Windows CI team
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: as above

File: doc/ci_admin_guide/Update_Zuul.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+## Steps
+
+1.  Install required `apt` dependencies
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: Where?

File: doc/ci_admin_guide/Update_Zuul.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+    - where `YOUR_PRIVATE_KEY` is a path to your SSH private key
+
+1.  Run `setup-zuul-server.yml` playbook
+
+    ```bash
+    ansible-playbook -i inventory.prod --private-key=YOUR_PRIVATE_KEY setup-zuul-server.yml
+    ```
+
+    If a playbook run completed successfully, then output should be following:
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: > Verify that the run completed successfully. The output should be following and `failed` task count must equal zero.

File: doc/ci_admin_guide/Update_Zuul.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    ```bash
+    cd ~/
+    git clone https://github.com/Juniper/contrail-windows-ci.git
+    cd contrail-windows-ci
+    ```
+
+1.  Make sure `development` branch contains correct PRs
+
+    ```bash
+    git log  # verify commit log
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: Make sure
verify

^ those are too broad, I wouldn't know what to look for

**Review by m-kostrzewa — APPROVED** [review]

File: doc/ci_admin_guide/Update_Zuul.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+    ```bash
+    cd ansible
+    ```
+
+1.  Create a virtualenv and install `pip` dependencies
+
+    ```bash
+    python3 -m virtualenv -p /usr/bin/python3 venv
+    source venv/bin/active
```
Comment (jacek-izykowski) [inline doc/ci_admin_guide/Update_Zuul.md]: It's `activate`.

**Review by jacek-izykowski — CHANGES_REQUESTED** [review]

File: doc/ci_admin_guide/Update_Zuul.md (hunk 6 above)
Comment (m-kostrzewa) [reply doc/ci_admin_guide/Update_Zuul.md]: :ghost:

File: doc/ci_admin_guide/Update_Zuul.md [hunk 7]
```diff
@@ -6,14 +6,15 @@ This procedure updates configuration using `development` branch from [contrail-w
 ## Prerequisites
 
 - Ubuntu 16.04 or Windows Subsystem for Linux with Ubuntu
+    - It will serve as Ansible control maching
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: machine*

**Review by m-kostrzewa — APPROVED** [review]

**Review by jacek-izykowski — APPROVED** [review]

**Pull request merged**